somnio_starter/__brick__/{{project_name.snakeCase()}}/packages/app_config_repository/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: app_config_repository
+description: Dart package which manages the app configuration domain.
+publish_to: none
+
+environment:
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: 3.10.0
+
+dependencies:
+  cloud_firestore: ^4.3.0
```

Comment:
check dependencies here:
- not every library is needed
- increase version number